### PR TITLE
Edge. CIE-2799. Updated the migration section.

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

**Review by BeateRixen — DISMISSED** [review]

**Review by MWindel — DISMISSED** [review]

File: content/edge/migration-bundle/migration-107-to-109.md [hunk 1]
```diff
@@ -30,16 +30,21 @@ In your Edge 10.7 setup, you must back up the data for each tenant and docker co
...
 
 ```shell
 mongodump --db=management --out OUTPUT_DIRECTORY # OUTPUT_DIRECTORY specifies the location of the backup.
 mongodump --db=edge --out OUTPUT_DIRECTORY
 mongodump --db=docker --out OUTPUT_DIRECTORY # This only needs to be done if microservices are enabled on 10.7.
 ```
-2. Note down the device ID of your Edge 10.7 appliance available at: `/usr/edge/properties/edge-agent/device-id`
-3. Create a backup of the `/etc/opcua` directory.
-4. Create a backup of the `/var/lib/cumulocity-agent/credentials` file.
+3. Note down the device ID of your Edge 10.7 appliance available at: `/usr/edge/properties/edge-agent/device-id`
```
Comment (MWindel) [inline content/edge/migration-bundle/migration-107-to-109.md:45]: available at: > available at

Comment (sharath-jk) [reply content/edge/migration-bundle/migration-107-to-109.md:45]: Incorporated.

File: content/edge/migration-bundle/migration-107-to-109.md [hunk 2]
```diff
@@ -30,16 +30,21 @@ In your Edge 10.7 setup, you must back up the data for each tenant and docker co
...
 
 ```shell
 mongodump --db=management --out OUTPUT_DIRECTORY # OUTPUT_DIRECTORY specifies the location of the backup.
 mongodump --db=edge --out OUTPUT_DIRECTORY
 mongodump --db=docker --out OUTPUT_DIRECTORY # This only needs to be done if microservices are enabled on 10.7.
 ```
-2. Note down the device ID of your Edge 10.7 appliance available at: `/usr/edge/properties/edge-agent/device-id`
-3. Create a backup of the `/etc/opcua` directory.
-4. Create a backup of the `/var/lib/cumulocity-agent/credentials` file.
+3. Note down the device ID of your Edge 10.7 appliance available at`/usr/edge/properties/edge-agent/device-id`
```
Comment (MWindel) [inline content/edge/migration-bundle/migration-107-to-109.md:45]: available at`/usr/edge/properties/edge-agent/device-id >  missing space before code block

**Review by MWindel — DISMISSED** [review]

File: content/edge/migration-bundle/migration-107-to-109.md (hunk 2 above)
Comment (sharath-jk) [reply content/edge/migration-bundle/migration-107-to-109.md]: Added a space.

File: content/edge/migration-bundle/migration-107-to-109.md [hunk 3]
```diff
@@ -30,16 +30,21 @@ In your Edge 10.7 setup, you must back up the data for each tenant and docker co
...
 
 ```shell
 mongodump --db=management --out OUTPUT_DIRECTORY # OUTPUT_DIRECTORY specifies the location of the backup.
 mongodump --db=edge --out OUTPUT_DIRECTORY
 mongodump --db=docker --out OUTPUT_DIRECTORY # This only needs to be done if microservices are enabled on 10.7.
 ```
-2. Note down the device ID of your Edge 10.7 appliance available at: `/usr/edge/properties/edge-agent/device-id`
-3. Create a backup of the `/etc/opcua` directory.
-4. Create a backup of the `/var/lib/cumulocity-agent/credentials` file.
+3. Note down the device ID of your Edge 10.7 appliance available at `/usr/edge/properties/edge-agent/device-id`
```
Comment (BeateRixen) [inline content/edge/migration-bundle/migration-107-to-109.md:45]: Without a colon this row must end with a full stop.

File: content/edge/remote-connectivity-bundle/data-exchange-using-data-broker.md [hunk 4]
```diff
@@ -6,6 +6,10 @@ layout: redirect
 
 Data broker lets you upload the data to a {{< product-c8y-iot >}} tenant account selectively. Note that you must first create a {{< product-c8y-iot >}} tenant account.
 
+> **Important:** Data broker and Edge connectivity is compatible only with the current version and previous two releases of {{< product-c8y-iot >}}.
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/data-exchange-using-data-broker.md:9]: is > are compatible?

File: content/edge/migration-bundle/migration-107-to-109.md (hunk 3 above)
Comment (sharath-jk) [reply content/edge/migration-bundle/migration-107-to-109.md:45]: Done

File: content/edge/remote-connectivity-bundle/data-exchange-using-data-broker.md (hunk 4 above)
Comment (sharath-jk) [reply content/edge/remote-connectivity-bundle/data-exchange-using-data-broker.md:9]: Right. Done :).

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**